src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/model/Prisoner.kt

```diff
@@ -92,11 +92,11 @@ open class Prisoner {
   @Schema(description = "In prison cell location", example = "A-1-002")
   var cellLocation: String? = null
 
-  @Field(type = FieldType.Nested)
+  @Field(type = FieldType.Nested, includeInParent = true)
   @Schema(description = "Aliases Names and Details")
   var aliases: List<PrisonerAlias>? = null
 
-  @Field(type = FieldType.Nested)
+  @Field(type = FieldType.Nested, includeInParent = true)
```

Comment:
Yes, with the @nested objects `aliases` and `alerts` in order to search these it would require a nestedQuery() - as it did before this PR. This had the problem that it does not recognise a cross-field keyword search e.g. John and Smith, when John was in the main document and Smith was in the aliases. For the AND, OR, NOT cross-field searches, the includeInParent allows these searches to reference fields in the main doc and in nested objects. If you compare the queries for AND, OR and NOT before and after this PR, you can see how much more concise and easy to understand they become. Other searches - global search and prisoner match still operated with nested queries.